app.js

```diff
@@ -0,0 +1,66 @@
+var app = {
+    Models: {},
+    Collections: {},
+    Views: {}
+};
+
+app.Models.TodoItem = Backbone.Model.extend({
+    defaults: {
+        // title: "",
+        completed: false
+    }
+});
+
+app.Collections.TodoItems = Backbone.Collection.extend({
+    model: app.Models.TodoItem,
+    // localStorage: new Store("backbone-todo")
+})
+
+app.Views.TodoApp = Backbone.View.extend({
+    el: ".todo",
+    initialize: function() {
+        this.newTodoField = this.$(".todo__new-todo-field");
+        this.markAllCompleteField = this.$(".mark-all-complete");
+        this.totalCount = this.$(".total-count");
+        this.remainingCount = this.$(".remaining-count");
+        this.completeCount = this.$(".complete-count");
+    }
+});
+app.Views.TodoItem = Backbone.View.extend({
+    tagName: "li",
+    className: "todo-item",
+    template: Handlebars.compile($("#todo-item-template").html()),
+    render: function() {
+        this.$el.html(this.template(this.model.toJSON()))
+        return this.$el;
+    }
+});
+app.Views.TodoItems = Backbone.View.extend({
+    el: ".todo-items",
+    initialize: function() {
+        // this.listenTo(this.collection, "add", );
+        // this.listenTo(this.collection, "remove", );
+
+        // app.todoItems.fetch();
+        this.render();
+    },
+    render: function() {
+        this.$el.html('');  // clear children
+        
+        let todoViews = this.collection.map(todoItem => {
+            return (new app.Views.TodoItem({model: todoItem})).render();
```

Comment:
So this isn't exactly doing what you think it is. you are actually storing a bunch of html in `todoViews` because you are returning `.render()`.

Instead you can do something more like this...

```
<Clean up code here first>
this.childViews = [];
let $html;

this.collection.forEach(todoItem => {
  const view = new app.Views.TodoItem({ model: todoItem });
  this.childViews.push(view);
  this.$el.append(view.render());
});
```

This will allow you to clean everything up, keep reference to the newly created views (actual views, not the DOM elements). and then render them to the DOM.